### ISSUE TITLE
Install and run errors on Deno v1.11.0

### ISSUE THREAD

**Issue by disyam** [jurassiscripts/velociraptor#64]
deno version: 1.11.0
os: fedora 34

when i try to install with this command: 
```deno install -qAn vr https://deno.land/x/velociraptor@1.0.0/cli.ts```

or run script with ```vr run [script-name]```, i got this error:

```
error: TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
    if (high!.operator === comp || high!.operator === ecomp) {
              ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1744:15

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
    if (high!.operator === comp || high!.operator === ecomp) {
                                         ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1744:42

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
      (!low!.operator || low!.operator === comp) &&
             ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1751:14

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
      (!low!.operator || low!.operator === comp) &&
                              ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1751:31

TS2339 [ERROR]: Property 'semver' does not exist on type 'never'.
      ltefn(version, low!.semver)
                          ~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1752:27

TS2339 [ERROR]: Property 'operator' does not exist on type 'never'.
    } else if (low!.operator === ecomp && ltfn(version, low!.semver)) {
                    ~~~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1755:21

TS2339 [ERROR]: Property 'semver' does not exist on type 'never'.
    } else if (low!.operator === ecomp && ltfn(version, low!.semver)) {
                                                             ~~~~~~
    at https://deno.land/x/semver@v1.0.0/mod.ts:1755:62

Found 7 errors.
```

**Comment by domino2** [comment]
Same on macOS 11.3.1 Big Sur

**Comment by umbopepato** [comment]
Hey @disyam and @domino2, thanks for reporting this!
Both Velociraptor and Hatcher (used by vr) depend on this version of `semver` causing the errors. I've updated it here but we'll have to wait for them to do the same in order to release a new working version.

In the meantime, please install vr with the `--no-check` flag on Deno 1.11.0 to bypass those type checks:

```shell
$ deno install --no-check -qAn vr https://deno.land/x/velociraptor@1.0.0/cli.ts
```

**Comment by umbopepato** [comment]
This is now fixed in `v1.0.1`, run `vr upgrade` to upgrade (or see [Installation](https://velociraptor.run/docs/installation/)).
